node/modules/services.go

```diff
@@ -114,3 +117,8 @@ func NewLocalDiscovery(ds dtypes.MetadataDS) *discovery.Local {
 func RetrievalResolver(l *discovery.Local) retrievalmarket.PeerResolver {
 	return discovery.Multi(l)
 }
+
+func RandomBeacon() beacon.DrandBeacon {
+	fmt.Println("RANDOM BEACON!")
```

Comment:
```suggestion
```

Reply:
but we're excited about the random beacon